EasyPost.Tests/UserTest.cs

```diff
@@ -18,6 +19,24 @@ public void TestRetrieveSelf()
             Assert.AreEqual(user.id, user2.id);
         }
 
+        [TestMethod]
+        public void TestUpdateBrand()
+        {
+            User user = User.Retrieve();
+            Assert.IsNotNull(user.id);
+
+            const string color = "#000000";
```

Comment:
Let's use a color that's super random so we know this went through (AKA not a default white or black).

Reply:
Resolved in [8d459a7](https://github.com/EasyPost/easypost-csharp/pull/203/commits/8d459a71ab9649ee007be4f7891e1ca9c06b371e)